package.json

```diff
@@ -3,6 +3,7 @@
   "scripts": {
     "build": "preconstruct build",
     "build:docs": "yarn build && yarn workspace docs build",
+    "dev:docs": "yarn workspace docs start",
```

Comment:
btw, did you manage to get HMR working **inside** the docs package itself?
AFAIR it didn't work for me last time after switching to preconstruct, even though I made sure to `preconstruct dev`
Like, other packages were successfully reloaded on change, but not docs package files

Reply:
Wait, so HMR _in the docs_ works for you on all code except of the the own code of the actual docs app?

Reply:
Yeah, from within the docs last time I had HMR for instance on components, but not on docs pages

If I understand correctly it was because "docs" is excluded from preconstruct config (since it has its own build tool), but for some reason gatsby did not provide HMR 🤷‍♂️ 

(Unless I missed a step)